src/events/components/DetailView/AttendanceEvent.tsx

```diff
@@ -47,6 +55,14 @@ const AttendanceEvent: FC<IProps> = ({ eventId, eventTitle }) => {
   const cancellationDeadline = DateTime.fromISO(attendanceEvent.unattend_deadline);
   const showPayment = !attendanceEvent.is_on_waitlist && attendanceEvent.is_attendee && attendanceEvent.payment;
 
+  const options: IExtraOption[] = [
+    { value: -1, label: 'Velg ekstra' },
```

Comment:
Shouldn't this just be a placeholder, instead of an option?

Reply:
I originally had the same thought, but I ended up using an option for it considering the backend api allows for resetting your choice of extras. If it was a placeholder, you would be locked into choosing an a valid option after one is already chosen, even if the event didn't necessarily require you to do so.

Reply:
But then again, it might be confusing and I'm not even sure if any events have ever had optional extras. I guess placeholder is the way to go, but I'd like another confirmation before I change it.

Reply:
Don't think you should be able to reset your extras, even if it's possible in the API. I think that if an event would have an option from "None of the above", they should make it an explicit one.

Reply:
Yeah, that makes sense.